### DISCUSSION TITLE
怎么加载自己本地的3dtiles数据

### DISCUSSION THREAD

**Discussion by EnjiLi** [zouyaoji/vue-cesium#259] [Q&A]
![image](https://user-images.githubusercontent.com/56714299/185735106-22628fe0-1f1d-4670-8039-6634a7242c7d.png)
怎么把这种本地没发布的数据通过vue-cesium加载到网页

**Comment by zouyaoji** [comment]
测试的话可以自己放前端工程静态资源目录，一般是 public文件夹下。

应用时可以服务端，java node nginx等静态资源目录。

**Comment by EnjiLi** [comment]
首先感谢您的解答。另外，我可以进vue-cesium的交流群么，官网的这个qq点击后没反应






------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2022年8月20日(星期六) 下午4:11
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [zouyaoji/vue-cesium] 怎么加载自己本地的3dtiles数据 (Discussion #259)





 
测试的话可以自己放前端工程静态资源目录，一般是 public文件夹下。
 
应用时可以服务端，java node nginx等静态资源目录。

**Reply by zouyaoji** [reply]
哦 应该是bug了

群号是 16533444
自己申请哈

**Reply by alaxury** [reply]
加群，被群管拒绝，又加，又拒绝
